src/state/configureStore.dev.ts

```diff
@@ -0,0 +1,19 @@
+import { Store } from "redux";
+import { IApplicationState, rootReducer, rootSaga } from "./ducks/index";
+import sagaMiddleware from "state/middleware/saga";
+import { configureStore } from "@reduxjs/toolkit";
+
+export default function configureAppStore(
+  initialState: IApplicationState
+): Store<IApplicationState> {
+  const store = configureStore({
+    reducer: rootReducer,
+    middleware: [sagaMiddleware],
+    devTools: true,
+    preloadedState: initialState,
+  });
+
+  sagaMiddleware.run(rootSaga);
```

Comment:
as you are not using `toolkit` change this code back to simple redux and remove `@reduxjs/toolkit` from packages